language-server/modules/langserver-core/src/test/resources/completion/annotation/endpointAnnotation1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "position": {
+    "line": 3,
+    "character": 1
+  },
+  "source": "annotation/source/endpointAnnotation1.bal",
+  "items": [
+    {
+      "label":"swagger:ClientEndpoint",
+      "detail":"Annotation",
+      "sortText":"110",
+      "insertText":"swagger:ClientEndpoint",
+      "insertTextFormat":"Snippet"
+    }
+  ]
+}
```

Comment:
Shall we add a line feed here

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/10205